cardigan_new/stories/global/typography/typography.stories.tsx

```diff
@@ -0,0 +1,153 @@
+import { ReactElement } from 'react';
+import styled from 'styled-components';
+
+import { font } from '@weco/common/utils/classnames';
+import { fontSizesAtBreakpoints } from '@weco/common/views/themes/typography';
+import MoreLink from '@weco/content/components/MoreLink/MoreLink';
+import Table from '@weco/content/components/Table';
+
+const Font = styled.div`
+  padding: 10px 0 25px;
+  border-bottom: 1px solid ${props => props.theme.color('neutral.500')};
+`;
+
+const FontName = styled.h2.attrs({
+  className: font('intb', 6),
+})`
+  color: ${props => props.theme.color('accent.purple')};
+`;
+
+const TypographyScale = ({ fontFamily }) => {
+  const transpose = matrix => {
+    const [row] = matrix;
+    return row.map((value, column) => matrix.map(row => row[column]));
+  };
+
+  const cols = Object.entries(fontSizesAtBreakpoints).map(entry => {
+    const value = entry[1];
+    return Object.entries(value).map((e, index) => {
+      const v = e[1];
+
+      return (
+        <span key={index} style={{ fontSize: `${v}rem`, fontFamily }}>
+          {v}rem ({v * 16}px)
+        </span>
+      );
+    });
+  });
+
+  const rows = transpose(cols);
+  const rowsWithScaleNumbers: (string | ReactElement)[][] = rows
+    .map((row, index) => {
+      return [
+        <strong key={index} style={{ fontSize: '22px' }}>
+          {index}
+        </strong>,
+        ...row,
+      ];
+    })
+    .filter(Boolean);
+  const firstRow: (string | ReactElement)[][] = [
+    ['', 'Small', 'Medium', 'Large'],
+  ];
+  const rowsWithHeadings = firstRow.concat(rowsWithScaleNumbers);
+
+  return <Table hasRowHeaders={false} rows={rowsWithHeadings} />;
+};
+
+const sizes = [0, 1, 2, 3, 4, 5, 6];
+const fontFamilies = ['intr', 'intb', 'wb', 'lr'];
+
+const Typography = ({ text }) => {
+  return (
+    <div>
+      {fontFamilies.map(font => (
+        <div key={font}>
+          {sizes.map(size => (
+            <Font key={size}>
+              <FontName>
+                <code>
+                  font-{font} font-size-{size}
+                </code>
+              </FontName>
+              <p
+                className={`font-size-${size} font-${font}`}
+                style={{ marginBottom: 0 }}
+              >
+                {text}
+              </p>
+            </Font>
+          ))}
+        </div>
+      ))}
+    </div>
+  );
+};
+
+const Template = args => <Typography {...args} />;
+export const families = Template.bind({});
+families.args = {
+  text: 'Wellcome Collection',
+};
+
+const ScaleTemplate = args => <TypographyScale {...args} />;
+export const scale = ScaleTemplate.bind({});
+scale.args = {
+  fontFamily: 'Wellcome Bold Web',
+};
+
+const MiscTemplate = () => (
+  <>
+    <div>
+      <h2>More link</h2>
+      <MoreLink name="Full event details" url="#" />
```

Comment:
This is also happening on the current Cardigan so could be fixed separately but thought I'd raise that the button icon is off
<img width="305" alt="Screenshot 2024-10-15 at 17 36 50" src="https://github.com/user-attachments/assets/4ae6b369-295f-4f12-b703-11d1578554e6">



http://localhost:54248/?path=/story/global-typography--misc